pyproject.toml

```diff
@@ -44,7 +44,7 @@ dependencies = [
 	"rv-ltl ~= 0.1.0a1",
 	"scipy ~= 1.7",
 	"shapely ~= 2.0",
-	"trimesh >3.21.5, <4",
+	"trimesh ~=3.21, >3.21.5",
```

Comment:
This has the same problem as before -- it's not clear what the first version that works is (is it `3.21.6` or `3.21.5.post2` or something else?). The constraint should be of the form `>=X, <4`.